seqr/views/utils/anvil_metadata_utils.py

```diff
@@ -185,16 +186,16 @@ def parse_anvil_metadata(
             sample_ids.add(sample.sample_id)
 
     saved_variants_by_family = _get_parsed_saved_discovery_variants_by_family(
-        list(family_data_by_id.keys()), include_metadata, include_svs=include_svs, variant_json_fields=variant_json_fields,
+        list(family_data_by_id.keys()), bool(mme_value), include_svs, variant_json_fields, variant_attr_fields,
```

Comment:
Why is the value of `mme_value` responsible for whether or not we include `seqr_chosen_consequence` on the parsed saved variant?

Reply:
You could rename `include_metadata` to `include_mme_metadata` to be more explicit

Reply:
because those fields go together, and I was trying to reduce the number of configuration options wherever I could. I could add another boolean flag to the function and pass that as true whenever I pass mme_value, but that seemed more complex than it is worth